### PR TITLE
Use different names for client Spans and server Spans.

### PR DESCRIPTION
**Pull request by Vizerai**

### CODE REVIEW

File: opencensus/plugins/grpc/internal/client_filter.h [hunk 1]
```diff
@@ -65,6 +67,7 @@ class CensusClientCallData : public grpc::CallData {
   grpc_linked_mdelem tracing_bin_;
   // Client method.
   absl::string_view method_;
```
Comment (g-easy) [inline opencensus/plugins/grpc/internal/client_filter.h]: Can method_ be removed?

File: opencensus/plugins/grpc/internal/filter.h [hunk 2]
```diff
@@ -104,6 +104,22 @@ trace::Span SpanFromCensusContext(const census_context *ctxt);
 // Returns a string representation of the StatusCode enum.
 absl::string_view StatusCodeToString(grpc_status_code code);
 
+inline absl::string_view GetMethod(const grpc_slice *path) {
+  if (GRPC_SLICE_IS_EMPTY(*path)) {
+    return "";
+  } else {
```
Comment (g-easy) [inline opencensus/plugins/grpc/internal/filter.h]: De-indent after return.

File: opencensus/plugins/grpc/internal/filter.h [hunk 3]
```diff
@@ -104,6 +104,22 @@ trace::Span SpanFromCensusContext(const census_context *ctxt);
...
+    const char *str =
+        reinterpret_cast<const char *>(GRPC_SLICE_START_PTR(*path));
+    size_t size = GRPC_SLICE_LENGTH(*path);
+    // Check for leading '/' and trim it if present.
```
Comment (g-easy) [inline opencensus/plugins/grpc/internal/filter.h]: Is there a helper function for this in absl::string_view?

Comment (g-easy) [reply opencensus/plugins/grpc/internal/filter.h]: `absl::StripPrefix(absl::string_view(GRPC_PTR, GRPC_LEN), "/")`

File: opencensus/plugins/grpc/internal/filter.h (hunk 2 above)
Comment (Vizerai) [reply opencensus/plugins/grpc/internal/filter.h]: Done.

File: opencensus/plugins/grpc/internal/client_filter.h (hunk 1 above)
Comment (Vizerai) [reply opencensus/plugins/grpc/internal/client_filter.h]: Done.

File: opencensus/plugins/grpc/internal/client_filter.cc [hunk 4]
```diff
@@ -139,18 +139,15 @@ grpc_error *CensusClientCallData::Init(grpc_call_element *elem,
                                        const grpc_call_element_args *args) {
   path_ = grpc_slice_ref_internal(args->path);
   start_time_ = absl::Now();
-  const char *method_str =
-      GPR_SLICE_IS_EMPTY(path_)
-          ? ""
-          : reinterpret_cast<const char *>(GRPC_SLICE_START_PTR(path_));
-  method_ = absl::string_view(
-      method_str, GRPC_SLICE_IS_EMPTY(path_) ? 0 : GRPC_SLICE_LENGTH(path_));
+  absl::string_view method = GetMethod(&path_);
```
Comment (g-easy) [inline opencensus/plugins/grpc/internal/client_filter.cc]: You can inline this into the next line.

File: opencensus/plugins/grpc/internal/server_filter.cc [hunk 5]
```diff
@@ -94,13 +94,8 @@ void CensusServerCallData::OnDoneRecvInitialMetadataCb(void *user_data,
...
     FilterInitialMetadata(initial_metadata, &sml);
     calld->path_ = grpc_slice_ref_internal(sml.path);
-    const char *method_str = GRPC_SLICE_IS_EMPTY(calld->path_)
-                                 ? ""
-                                 : reinterpret_cast<const char *>(
-                                       GRPC_SLICE_START_PTR(calld->path_));
-    calld->method_ = absl::string_view(
-        method_str,
-        GRPC_SLICE_IS_EMPTY(sml.path) ? 0 : GRPC_SLICE_LENGTH(sml.path));
+    absl::string_view method = GetMethod(&calld->path_);
```
Comment (g-easy) [inline opencensus/plugins/grpc/internal/server_filter.cc]: Ditto.

**Pull request merged**